Update README.md to better explain Syntax Theme installation (#225)
StylishThemes/StackOverflow-Dark: README.md

📦 [Install the usercss](https://github.com/StylishThemes/StackOverflow-Dark/raw/master/stackoverflow-dark.user.css). Supports automatic updates.

> ## Additional styles (required for properly rendering blocks of code on StackOverflow pages)

Reviewer: ```suggestion
## Additional styles
```

> ## Additional styles (required for properly rendering blocks of code on StackOverflow pages)

Reviewer: ```suggestion
## Additional styles
```

Not all additional styles will be required. This one is thats it.

Author: Yes, that is correct. Didn't look at it that way. :+1:

> 

Reviewer: ```suggestion
(required for properly rendering blocks of code on StackOverflow pages)
```

Author: Yes, this is okay with me.

## Additional styles

📦 [Install the StackOverflow Syntax Themes usercss](https://raw.githubusercontent.com/StylishThemes/Feature-Override-Styles/master/stackoverflow-syntax-themes.user.css)<br>

> - This style allows you to choose preset syntax themes for StackOverflow and associated sites.

Reviewer: ```suggestion

```
I forgot about this sentence. Needs to be shortened the whole lot I think. Less reading better for all users apparently.

> - The StackOverflow Syntax Themes style is **a required additional style** for properly rendering blocks of code on StackOverflow pages<br><br>

Reviewer: ```suggestion
- The StackOverflow Syntax Themes style is **a required additional style** for rendering blocks of code on StackOverflow pages it allows you to choose preset syntax themes.<br><br>
```
